### PR TITLE
Fixed some gomnd and wsl linting for v2

### PR DESCRIPTION
**Pull request by lonnblad**

### CODE REVIEW

File: v2/datadog/aws-cloudwatch-log-events/cloudwatch_test.go [hunk 1]
```diff
@@ -83,8 +84,8 @@ var expectedDatadogLogs = []interface{}{
 		"message":          "msg",
 		"service":          "keckebarn",
 		"timestamp":        validLambdaLogEvent.LogEvents[1].Timestamp,
-		"dd.trace_id":      uint64(123),
-		"dd.span_id":       uint64(456),
+		"dd.trace_id":      uint64(123), // nolint: gomnd
```
Comment (rickardenglund) [inline v2/datadog/aws-cloudwatch-log-events/cloudwatch_test.go]: Why did you not use a const trace_id = 123 here?

File: v2/array/array.go [hunk 2]
```diff
@@ -41,10 +43,13 @@ func IntersectString(arrs ...[]string) (result []string) {
...
 }
 
+const oneOccurence = 1
+
 func DifferenceString(arrs ...[]string) (result []string) {
 	result = []string{}
+
 	for key, count := range countMap(arrs...) {
-		if count == 1 {
+		if count == oneOccurence {
```
Comment (rickardenglund) [inline v2/array/array.go]: Are you sure that this is needed? it is not complaining on the one i v2/datadog/aws-cloudwatch-log-events/cloudwatch_test.go.

In this case I would say that a nolint would be clearer.

File: v2/timeutils/milli_test.go [hunk 3]
```diff
@@ -8,13 +8,13 @@ import (
 )
 
 func Test_MillisecondsShouldBeWithinRealisticInterval(t *testing.T) {
-	assert.True(t, MillisecondsNow() > 1528030261000) // 2018-03-06
-	assert.True(t, MillisecondsNow() < 9999999999999) // 2286-11-20
+	assert.True(t, MillisecondsNow() > 1528030261000) // nolint: gomnd
```
Comment (rickardenglund) [inline v2/timeutils/milli_test.go]: This number I actually don't know.
And the other one I think should maxInt?

File: v2/timeutils/milli_test.go [hunk 4]
```diff
@@ -8,13 +8,13 @@ import (
...
+	assert.True(t, MillisecondsNow() < 9999999999999) // nolint: gomnd
 }
 
 func Test_MillisecondsUnix(t *testing.T) {
 	now := time.Now()
-	assert.Equal(t, MillisecondsUnix(now.Add(time.Second)), MillisecondsUnix(now)+1000)
+	assert.Equal(t, MillisecondsUnix(now.Add(time.Second)), MillisecondsUnix(now)+1000) // nolint: gomnd
```
Comment (rickardenglund) [inline v2/timeutils/milli_test.go]: + 1 * time.second?

File: v2/datadog/aws-cloudwatch-log-events/cloudwatch_test.go (hunk 1 above)
Comment (lonnblad) [reply v2/datadog/aws-cloudwatch-log-events/cloudwatch_test.go]: don't believe it adds any value, it is just a magic number used in a test

File: v2/array/array.go (hunk 2 above)
Comment (lonnblad) [reply v2/array/array.go]: the linter failed for this

how can a nolint be clearer?

File: v2/timeutils/milli_test.go (hunk 3 above)
Comment (lonnblad) [reply v2/timeutils/milli_test.go]: I took them as magic numbers used in a test.

File: v2/timeutils/milli_test.go (hunk 4 above)
Comment (lonnblad) [reply v2/timeutils/milli_test.go]: Where do you believe I should add or change to `1 * time.Second`?

File: v2/array/array.go (hunk 2 above)
Comment (rickardenglund) [reply v2/array/array.go]: I get suspicious when I read the oneOccurence thing and feel the urge to check that it is really a 1.

File: v2/timeutils/milli_test.go (hunk 3 above)
Comment (rickardenglund) [reply v2/timeutils/milli_test.go]: But you removed the translation of the numbers? in the old test It was easy to see what dates we were actually comparing to. Now I need to translate the number to find out what realistic Intervall means.

File: v2/timeutils/milli_test.go (hunk 4 above)
Comment (rickardenglund) [reply v2/timeutils/milli_test.go]: instead of +1000 but I guess the linter would complain on the one. I think your solution is good here.

**Pull request merged**